doc/source/rllib-algorithms.rst

```diff
@@ -302,8 +304,14 @@ SpaceInvaders  650                       1001                           1025
 
 Policy Gradients
 ----------------
-|pytorch| |tensorflow|
-`[paper] <https://papers.nips.cc/paper/1713-policy-gradient-methods-for-reinforcement-learning-with-function-approximation.pdf>`__ `[implementation] <https://github.com/ray-project/ray/blob/master/rllib/agents/pg/pg.py>`__ We include a vanilla policy gradients implementation as an example algorithm.
+
+**Frameworks:** PyTorch |pytorch|, TensorFlow |tensorflow|
+
+**Paper:** `Policy Gradient Methods for Reinforcement Learning with Function Approximation <https://papers.nips.cc/paper/1713-policy-gradient-methods-for-reinforcement-learning-with-function-approximation.pdf>`__ `[implementation] <https://github.com/ray-project/ray/blob/master/rllib/agents/pg/pg.py>`__
+
+**Implementation:** We include a `vanilla policy gradients implementation <https://github.com/ray-project/ray/blob/master/rllib/agents/pg/pg.py>`__ as an example algorithm.
```

Comment:
I think the original formatting is better; this is too verbose.

Reply:
done